ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -78,10 +79,10 @@
                     this.widgetObject = new DatePickerWidget(this, this.getWidget(), model);
                 }
                 if (this.widgetObject.getValue() !== '') {
-                    this._model.value = this.widgetObject.getValue();
+                    super.updateFormattedDate(this.widgetObject.getValue());
```

Comment:
I would like to keep the changes simple here, no new class for now
```
if (editFormat) {
   this._model.value = parse(...,...);
} else {
   this._model.value = this.widgetObject.getValue()
}
```

And fix af2-web-runtime to allow setting date objects in value.